### ISSUE TITLE
An ignored/excluded file, still gets added to the graph when edited and saved

### ISSUE THREAD

**Issue by ribeirompl** [foambubble/foam#480]
- Foam version: v0.9.1
- Platform: Linux
- Issue occur on the [foam template](https://github.com/foambubble/foam-template) repo: No

**Summary**
Within `.vscode/settings.json`, I added the following line:
`"foam.files.ignore": ["readme.md"]`

I reloaded vscode, and that file no longer was on the graph, as expected.
However when I made a change to the readme and then saved the file, foam added the file back onto the graph.
Only after reloading vscode, did the file no longer appear on the graph. However, editing the readme would add it back.

So it appears that when a markdown file is saved, that the graph does not check the ignore list, before updating itself?

**Steps to reproduce**
1. Clone the [foam template](https://github.com/foambubble/foam-template) repo (although any foam notes repo should work).
2. Confirm that `readme.md` appears on the graph. For the foam-template repo, it is named "Foam" on the graph.
3. Add `"foam.files.ignore": ["readme.md"]` to `settings.json`.
4. Note, that closing and reopening the graph, does not remove the readme file.
5. Reload vscode, and confirm that the readme now does not appear on the graph.
6. Edit `readme.md`, and save the file.
7. Look at the graph to see the readme appear (even though it should be ignored).

**Comment by riccardoferretti** [comment]
Fixed in 0.17.8

**Comment by pallavbakshi** [comment]
The issue still exists.

**Comment by riccardoferretti** [comment]
@pallavbakshi I can't reproduce it following the steps above, can you please create a new issue with the steps to reproduce the problem?
